pipeline/trusted-resources/pkg/config/config.go

```diff
@@ -31,23 +31,24 @@ type Config struct {
 
 const (
 	// cosignKey is the name of consign configmap
```

Comment:
nit: CosignKey, so as the following two const vars

Reply:
thanks I'm revising these 😂